_includes/JasmineBio.html

```diff
@@ -0,0 +1,235 @@
+<html>
+
+<head>
+<title>Jasmine Liu</title>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"/>
```

Comment:
If you are going to use the node_modules version of animate.css, you don't need to include the CDN version of this. Just choose one